lib/app/modules/core/capability_statement_sequence.rb

```diff
@@ -71,9 +71,22 @@ class CapabilityStatementSequence < SequenceBase
         end
       end
 
-      test 'FHIR server supports the conformance interaction that defines how it supports resources' do
+      test 'Test FHIR version' do
         metadata do
           id '02'
+          link 'http://www.hl7.org/fhir/directory.cfml'
+          desc %(
+            Testing that the FHIR client is the same version that was version as the testing instance that was selected.
```

Comment:
The existing sentence doesn't really make sense.  Can we make this a little clearer and detailed?
```suggestion
            Checks that the FHIR version of the server matches the FHIR version expected by the tests.
            This test will inspect the CapabilityStatement returned by the server to verify the FHIR version of the server.
```